tox.ini

```diff
@@ -33,10 +33,11 @@ description = Check code against coding style standards
 deps =
     black
     flake8<6.0.0
+    flake8-builtins
+    flake8-copyright
     flake8-docstrings>=1.6.0
     flake8-docstrings-complete>=1.0.3
-    flake8-copyright
-    flake8-builtins
+    flake8-test-docs
```

Comment:
Can you order all of the deps alphabetically?

Reply:
Good idea! Will do the same for other dependencies as well :) thanks!